test/nested_suite_test/main_suite.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<suites version="2.0">
+
+  <suite name="radiation3_suite">
+    <group name="rad_lw_group">
+      <scheme>rad_lw</scheme>
+    </group>
+    <group name="rad_sw_group">
+      <nested_suite name="radiation3_subsuite" group="rad_sw_group" file="radiation3_subsuite.xml"/>
+    </group>
+  </suite>
+
+  <suite name="main_suite">
+    <group name="radiation1">
+      <subcycle loop="num_subcycles_for_effr">
+        <scheme>effr_pre</scheme>
+        <subcycle loop="2">
+          <subcycle loop="2">
```

Comment:
Was this nested subcycle intentional?

Reply:
Yes, that is from the test that I used as a template for this new test.